src/App.tsx

```diff
@@ -1,18 +1,34 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [selectValue, setSelectValue] = useState(5);
```

Comment:
Small comment - it would be nice to create an array of possible perPage options, and also use it to render options in select 
For example:
`const PER_PAGE_SELECTOR = [3, 5, 10, 20]
`